### PR TITLE
:sparkles:  Feat(repots): Market export V5

### PR DESCRIPTION
**Pull request by Jkaiser001**
# Story:

> Update docs with new endpoint Market Data V5
## Fix
- Update the changes in the different APi endpoints
- Add US country
- fix some texts

### [Stage](https://stage.admetricks.com/docs/#datos-de-mercado-v5-0)

### CODE REVIEW

File: source/includes/_resources.md.erb [hunk 1]
```diff
@@ -80,6 +80,7 @@ y los valores que puedes utilizar son los siguientes:
 | 17  | El Salvador          |
 | 18  | República Dominicana |
 | 19  | Paraguay             |
+| 30  | Estados Unidos       |
```
Comment (Jkaiser001) [inline source/includes/_resources.md.erb:83]: new country

File: source/includes/snippets/_post-datos-de-mercado-(v4.0).md [hunk 2]
```diff
@@ -0,0 +1,146 @@
+> Ejemplo de petición
```
Comment (Jkaiser001) [inline source/includes/snippets/_post-datos-de-mercado-(v4.0).md:1]: Snippet Market Data V4

File: source/includes/snippets/_post-datos-de-mercado-(v5.0).md [hunk 3]
```diff
@@ -0,0 +1,159 @@
+> Ejemplo de petición
```
Comment (Jkaiser001) [inline source/includes/snippets/_post-datos-de-mercado-(v5.0).md:1]: Snippet Market Data V5

File: source/includes/api.html.md.erb [hunk 4]
```diff
@@ -206,6 +211,11 @@ Entrega una lista de marcas filtradas (opcionalmente) por nombre y país.
         <small class="param__type">string</small>
         
         
+          <span class="param__desc">Fitrar por país</span>
```
Comment (vitorohe) [inline source/includes/api.html.md.erb]:
```suggestion
          <span class="param__desc">Filtrar por país</span>
```

File: source/includes/api.html.md.erb [hunk 5]
```diff
@@ -64,6 +64,11 @@ Entrega una lista de anuncios filtrados por país.
...
```
Comment (vitorohe) [inline source/includes/api.html.md.erb]:
```suggestion
          <span class="param__desc">Filtrar por país</span>
```

File: source/includes/api.html.md.erb [hunk 6]
```diff
@@ -1261,6 +1292,11 @@ Entrega una lista de industrias filtradas por nombre.
...
```
Comment (vitorohe) [inline source/includes/api.html.md.erb]:
```suggestion
          <span class="param__desc">Filtrar por país</span>
```

File: source/includes/api.html.md.erb [hunk 7]
```diff
@@ -2346,6 +2408,11 @@ Entrega una lista de sitios web filtrados (opcionalmente) por nombre y país.
...
```
Comment (vitorohe) [inline source/includes/api.html.md.erb]:
```suggestion
          <span class="param__desc">Filtrar por país</span>
```

File: source/includes/api.html.md.erb [hunk 8]
```diff
@@ -1243,7 +1274,7 @@ Para realizar esta operación es obligatorio utilizar tu <strong><a href="#confi
   
 </ul>
 
-Entrega una lista de industrias filtradas por nombre.
+Entrega una lista de industrias filtradas por nombre del idioma del usuario.
```
Comment (vitorohe) [inline source/includes/api.html.md.erb]: are you sure they are filtered by language?

Comment (Jkaiser001) [reply source/includes/api.html.md.erb]: In a Cristian commit on clientela changes this line https://github.com/admetricks/clientela/commit/bad2a127d27aa3f1a9040f1d2bbf42c19c602f0b

Comment (Jkaiser001) [reply source/includes/api.html.md.erb]: image

Comment (Jkaiser001) [reply source/includes/api.html.md.erb]: @vitorohe

**Review by pdelsol — APPROVED** [review]
yo lo veo bien, por lo menos de market v5, las otros cambios no cache

File: source/includes/api.html.md.erb (hunk 8 above)
Comment (vitorohe) [reply source/includes/api.html.md.erb]: so, industries come in the language of the user?
```
Entrega una lista de industrias filtradas por nombre en el idioma del usuario
```

Comment (Jkaiser001) [reply source/includes/api.html.md.erb]: I think, yes 
image

**Review by vitorohe — APPROVED** [review]
LGTM

**Pull request merged**